src/cascadia/LocalTests_TerminalApp/SettingsTests.cpp

```diff
@@ -1366,4 +1369,37 @@ namespace TerminalAppLocalTests
             VERIFY_ARE_EQUAL(guid3, settings._globals._defaultProfile);
         }
     }
+    void SettingsTests::TestProfileIconWithEnvVar()
+    {
+        const std::string settingsJson{ R"(
+        {
+            "profiles": [
+                {
+                    "name": "profile0",
+                    "icon": "%WINDIR%\\System32\\x_80.png"
+                }
+            ]
+        })" };
+
+        VerifyParseSucceeded(settingsJson);
+        CascadiaSettings settings;
+        settings._ParseJsonString(settingsJson, false);

```

Comment:
these tests don't actually test that the value got turned into something

Reply:
Why a new PR? Sometimes it's just quicker to kill & recreate rather than perform surgery ;)

Agree re. tests. Will update them tomorrow.